recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   sha256: {{ sha256 }}
   patches:
     - llvm-lto-static.patch   # [win]
-    - partial-testing.patch
+    - partial-testing.patch   # [not win]
```

Comment:
Can you revert this change now that azure works and timing out is not an issue?

Reply:
Thanks for your help Isuru! Honestly, this patch look unnecessary without calls for llvm-lit which I also had to comment out. I'll rather comment it out completely with a TODO note